package.json

```diff
@@ -42,6 +42,7 @@
     "typescript": "^4.7.3"
   },
   "devDependencies": {
+    "@arbitrum/sdk": "^3.1.2",
```

Comment:
would be good to update the package name above `"name": "@uniswap/governance-seatbelt",` to `@arbitrum`